recipes/searchgui/meta.yaml

```diff
@@ -1,25 +1,27 @@
 # Do not forget to update the version string in the searchgui.py file
 {% set name = "SearchGUI" %}
-{% set version = "4.0.1.alpha" %}
+{% set version = "4.0.0" %}
```

Comment:
this will cause problems, as the latest version will still be the 4.0.1

Reply:
Um...ok, then I will update the app to last bugfix version 4.0.3...